app/components/HeaderBabyMenu.tsx

```diff
@@ -209,7 +209,7 @@ const HeaderBabyMenu = (props: any): any => {
         ) : (
           <ProfileListView>
             <ProfileIconView>
-              {data.photoUri != '' ? (
+              {(data.photoUri != '' && data.photoUri != null) ? (

```

Comment:
If you want to also make sure photoUri is not undefined then use
`{ data.photoUri ? (`
instead of 
`{(data.photoUri != '' && data.photoUri != null) ? (`